login-workflow/src/new-architecture/contexts/sharedDictionaries/types.ts

```diff
@@ -0,0 +1,68 @@
+export type CommonTranslations = {
```

Comment:
@JeffGreiner-eaton @daileytj @manojleaton @surajeaton since I moved common translations to another folder. Most of the current translations will break.

I will raise another PR for fixing all transactions because already this PR has so many changes to review and test

Reply:
Will keep bluiCommon namespace.